app/src/main/java/com/hedvig/app/feature/ratings/RatingsDialog.kt

```diff
@@ -64,7 +65,11 @@ class RatingsDialog : DialogFragment() {
                 when (choice) {
                     RatingsChoice.YES -> {
                         tracker.rate()
```

Comment:
Can probably extract this as a function to Util as well, like `tryOpenPlayStore()` or something

Reply:
Agreed. I'll do that instead I think